adls-plugins/docs/ADLSBatchSink-batchsink.md

```diff
@@ -10,6 +10,19 @@ Properties
 
 **path** Path to directory to store output files. The path must start with `adl://`
```

Comment:
please update the existing properties to conform to the new ones you added, with capitalization and spaces between words.